Corona-Warn-App/src/main/java/de/rki/coronawarnapp/tracing/ui/details/items/riskdetails/DetailsLowRiskBox.kt

```diff
@@ -32,11 +30,6 @@ class DetailsLowRiskBox(
         payloads: List<Any>
     ) -> Unit = { item, _ ->
         info = item
-        riskDetailsInformationLowriskBodyUrl.convertToHyperlink(
-            context.getString(R.string.risk_details_explanation_faq_link)
```

Comment:
I think the corresponding string resource can be removed as it doesn't seems to be reused somewhere else.

Reply:
Removed